### PR TITLE
Add Prod Profile to GAE Deployment Command

### PR DESCRIPTION
**Pull request by SudharakaP**
Fixes https://github.com/jhipster/generator-jhipster/issues/11253

-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by SudharakaP** [comment]
Closing this since it was decided to add the prod profile instead; and the webpack profile is not recommended for production.

**Comment by pascalgrimaud** [comment]
it looks like the cassandra failures are back...? so sad...!

**Comment by SudharakaP** [comment]
@pascalgrimaud : We shall revert the removal of cassandra unit back and see how it goes. I have to investigate more. :thinking:

File: generators/gae/index.js [hunk 1]
```diff
@@ -773,7 +773,7 @@ module.exports = class extends BaseGenerator {
 
                 if (this.buildTool === 'maven') {
                     this.log(chalk.bold('\nRun App Engine DevServer Locally: ./mvnw package appengine:run -DskipTests'));
-                    this.log(chalk.bold('Deploy to App Engine: ./mvnw package appengine:deploy -DskipTests -Pgae,prod-gae'));
+                    this.log(chalk.bold('Deploy to App Engine: ./mvnw package appengine:deploy -DskipTests -Pgae,prod,prod-gae'));
                 } else if (this.buildTool === 'gradle') {
                     this.log(chalk.bold('\nRun App Engine DevServer Locally: ./gradlew appengineRun'));
                     this.log(chalk.bold('Deploy to App Engine: ./gradlew appengineDeploy -Pgae -Pprod-gae'));
```
Comment (ludoch) [inline generators/gae/index.js]: Maybe add -Pprod for Gradle as well?

Comment (SudharakaP) [reply generators/gae/index.js]: @ludoch : I thought about that just for consistency sake, but it's not needed for gradle as the change (https://github.com/jhipster/generator-jhipster/pull/11207) only effect maven. Do you think we should add it just for make the two commands consistent? :smile:

**Pull request merged**